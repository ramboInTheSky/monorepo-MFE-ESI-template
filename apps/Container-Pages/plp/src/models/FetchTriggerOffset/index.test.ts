import {FetchTriggerOffset} from "."

describe("Model - FetchTriggerOffset: ", () => {
    const mockTestState: FetchTriggerOffset = {
        inPageFiltersEnabled: {
            xs: 4,
            sm: 4,
            md: 4,
            lg: 6,
            xl: 8,
        },
        default: {
            xs: 4,
            sm: 4,
            md: 6,
            lg: 6,
            xl: 8,
        },
    }
    it("should match the FetchTriggerOffset", () => {
        expect(mockTestState).toMatchSnapshot()
    })
})
