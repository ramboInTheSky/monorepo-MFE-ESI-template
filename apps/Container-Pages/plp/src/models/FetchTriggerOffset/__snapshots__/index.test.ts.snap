// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model - FetchTriggerOffset:  should match the FetchTriggerOffset 1`] = `
Object {
  "default": Object {
    "lg": 6,
    "md": 6,
    "sm": 4,
    "xl": 8,
    "xs": 4,
  },
  "inPageFiltersEnabled": Object {
    "lg": 6,
    "md": 4,
    "sm": 4,
    "xl": 8,
    "xs": 4,
  },
}
`;
